datalad/core/local/status.py

```diff
@@ -495,3 +495,24 @@ def custom_result_summary_renderer(results):  # pragma: more cover
                for r in results):
             from datalad.ui import ui
             ui.message("nothing to save, working tree clean")
+
+        # ### Report on possibly cleanable temporary locations
+
+        # Infer the dataset we ran on from the results and get original
+        # recursive (+limit) options:
+        refds = set(r['refds'] for r in results)
+        assert len(refds) == 1
+        refds = Dataset(refds.pop())
+        recursive = kwargs.pop('recursive', False)
+        recursion_limit = kwargs.pop('recursion_limit', None)
+        clean_infos = refds.clean(info=True,
+                                  recursive=recursive,
+                                  recursion_limit=recursion_limit,
```

Comment:
I admittedly did not think about recursion until now. But maybe we should:

```
/ds/hcp/super (git)-[master] % datalad clean -r 
action summary:
  clean (notneeded: 18000)
datalad clean -r  22.20s user 13.67s system 22% cpu 2:38.55 total
```

I think this calls for some kind of switch/config setting, or we decide that switching the renderer to something else is good enough. However, worth acknowledging explicitly one way or another.

Reply:
I wondered about that, too. If the reporting should become configurable the "problem" appears to be the default. On the one hand: That hint is kinda important to be there by default - if you know you want to enable it, you probably don't really need it. On the other hand: Being more expensive by default is kinda wrong, too.